accounts/models.py

```diff
@@ -1,25 +1,58 @@
 import datetime
 from django.db import models
-from django.contrib.auth.models import User
+from django.contrib.auth.models import BaseUserManager, AbstractBaseUser
 from django.utils import timezone
 
 
-class User(models.Model):
+
+class UserManager(BaseUserManager):
+
+
+  def _create_user(self, email, password, active, staff, superuser,
+                   **extra_fields):
+    now = timezone.now()
+    email = self.normalize_email(email)
+    user = self.model(email=email, staff=staff, active=active,
+                      superuser=superuser, register_date=now, **extra_fields)
+    user.set_password(password)
+    user.save(using=self._db)
+    return user
+
+  def create_user(self, email=None, password=None, **extra_fields):
+    return self._create_user(email, password, False, False, False,
+                 **extra_fields)
+
+  def create_superuser(self, email, password, **extra_fields):
+    user=self._create_user(email, password, True, True, True, **extra_fields)
+    user.save(using=self._db)
+    return user
+
+
+class User(AbstractBaseUser):
 
     GENDER = ((1, 'Female'), (2, 'Male'))
 
     email = models.EmailField(max_length=100, unique=True)
-    password = models.CharField(max_length=20)
     name = models.CharField(max_length=50)
     surname = models.CharField(max_length=50)
-    gender = models.IntegerField(choices=GENDER)
+    gender = models.IntegerField(choices=GENDER, default=1)
     telephone_number = models.CharField(max_length=20)
     address = models.CharField(max_length=200)
     register_date = models.DateTimeField(auto_now=False, auto_now_add=True)
     register_updated = models.DateTimeField(auto_now=True, auto_now_add=False)
     activation_key = models.CharField(max_length=40, default='')
-    akey_expires = models.DateTimeField(default=timezone.now() + datetime.timedelta(2))
-    is_active = models.BooleanField(default=False)
+    date_of_birth = models.DateTimeField(default=timezone.now())
+    akey_expires = models.DateTimeField(default=timezone.now() +
```

Comment:
No chyba żartujesz. Przecież to jeden z najbardziej podstawowych błędów na jakie nacinają się nowicjusze w Django. Zresztą pole wyżej to samo jest.

jeżeli robimy po prostu ```default=timezone.now()``` to ```timezone.now()``` zostanie wykonane podczas importu pliku models.py a do defaultu pola trafi już konkretna wartość datetime - z czasu importu models.py.

Django pozwala na przekazanie funkcji do default. Jeżeli default to funkcja to django będzie ją wykonywało za kazdym razem gdy potrzebuje defaulta. Dlatego do default nie przekazujemy ```timezone.now()``` tylko ```timezone.now```. I dlatego W przypadku akey_expires trzeba użyć lambdy.

Jeżeli to Ci nie wystarcza:
http://stackoverflow.com/questions/13225890/django-default-timezone-now-saves-records-using-old-time
http://stackoverflow.com/questions/2771676/django-default-datetime-now-problem
http://david.feinzeig.com/blog/2011/12/06/how-to-properly-set-a-default-value-for-a-datetimefield-in-django/
http://atodorov.org/blog/2014/04/15/beware-of-django-default-model-field-option-when-using-datetime-now/